darts/tests/test_timeseries_generation.py

```diff
@@ -75,3 +77,20 @@ def test_routine(length):
 
         for length in [1, 2, 5, 10, 100]:
             test_routine(length)
+
+    def test_holidays_timeseries(self):
+        time_index_1 = pd.date_range(periods=365*3, freq='D', start=pd.Timestamp('2012-01-01'))
+        time_index_2 = pd.date_range(periods=365 * 3, freq='D', start=pd.Timestamp('2014-12-24'))
+        time_index_3 = pd.date_range(periods=10, freq='Y', start=pd.Timestamp('1950-01-01')) + pd.Timedelta(days=1)
+
+        # testing we have at least one holiday flag in each year
+        def test_routine(time_index, country_code):
+            ts = holidays_timeseries(time_index, country_code)
+            self.assertTrue(
+                all(ts._df.groupby(pd.Grouper(freq="y")).sum().values)
```

Comment:
it'd be slightly better to call `pd_dataframe()` instead of the private `_df`.

Reply:
You are right. I have changed it!